app/components/job-schedule/details/job-schedule-configuration.component.ts

```diff
@@ -22,8 +24,26 @@ export class JobScheduleConfigurationComponent {
     public executionInfo: any = {};
     public schedule: any = {};
     public jobScheduleMetadata: List<Metadata> = List([]);
+    public jsonViewEnabled = true;
 
     private _jobSchedule: JobSchedule;
+    private _sub: Subscription;
+
+    constructor(
+        private changeDetector: ChangeDetectorRef,
+        private workspaceService: WorkspaceService) {
+
+        this._sub = this.workspaceService.currentWorkspace.subscribe((ws) => {
+            if (ws) {
```

Comment:
i feel like we should always assume this is never null. We'll make sure there is always a value in the service